node/flatpak_node_generator/providers/npm.py

```diff
@@ -19,13 +19,16 @@
 import textwrap
 import types
 
+from yarl import URL
```

Comment:
Any reason we can't use urllib here? (If that's not workable it's fine too, but then this should be in the `pyproject.toml`.)

Reply:
I just like `yarl` more for the pathlib-like `URL() / ""` syntax. But actually, it's not really beneficial here; will replace with `urllib`.

Reply:
(TBH if you have package suggestions in general, I'd be open to hear them, since `pipx` == actually safe installs now.)

Reply:
Done.